resources/migrations/000_migrations.yaml

```diff
@@ -3646,3 +3646,51 @@ databaseChangeLog:
             columns:
               - column:
                   name: dashboard_id
+  - changeSet:
+      id: 56
+      author: senior
+      changes:
+        - createTable:
+            tableName: metabase_dimensions
```

Comment:
This table name really doesn't need a `metabase_` prefix. The only reason some of our tables have that is for historic reasons -- the original version of Metabase was developed using Django which adds a prefix to DB table names depending on which module they belong to (this is also why some of our tables are prefixed with `report_`)